public/config/settings.js

```diff
@@ -386,41 +337,51 @@ options.mappings = [{
 	// Represents player crafting
 	game  : "Fallout New Vegas",
 	type  : "craft",
+	actions : ["CraftingTable"],
 	columns : ["action", "name", "playerID", "value"],
 }, {
 
 	// Represents player getting attacked by giant rat
 	game  : "Fallout New Vegas",
 	type  : "ratattack",
+	actions : ["Creature Giant Rat attacked first"],
 	columns : ["action", "playerID", "value"],
 }, {
 
 	// Represents player getting some object
 	game  : "Fallout New Vegas",
 	type  : "object",
+	actions : ["ObjectOnActivate"],
 	columns : ["action", "????", "value"],
 }, {
 
 	// Represents player doing some action
 	game  : "Fallout New Vegas",
 	type  : "playeraction",
-	columns : ["action", "playerID", "value", "object"],
+	actions : [ "player attacked first", "Player killed",
+				"Player looted dead", "Player looted Dead", "Player shooting a dead"],
+	columns : ["action", "playerID", "value", "object"]
 }, {
 
 	// Represents player on quest
 	game  : "Fallout New Vegas",
 	type  : "quest",
+	actions : ["Quest"],
 	columns : ["action", "playerID", "value", "name", "status"],
 }, {
 
 	// Represents player stat
 	game  : "Fallout New Vegas",
 	type  : "stat",
+	actions : ["Stat"],
 	columns : ["action", "playerID", "action", "key", "value", "status"],
 },
 
 {
 	game : "Game Gaze",
 	type : "default",
+	actions : ["Game Gaze"],
```

Comment:
In the case that a game does not have a first "action indicator" column, how do we want to represent that in the Key Mapping? We could set the 'actions' key to *null*?. I want to make sure that the end user can distinguish between key mapping that have actions, vs key mappings that represent games with only one type of action.